app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/UITestHelper.java

```diff
@@ -99,4 +130,42 @@ private static Bitmap getBitmap(Drawable drawable){
         drawable.draw(canvas);
         return bitmap;
     }
+
+    /**
+     * Create a file with the temp_diablerets as image, given a filename
+     * @param filename the filename
+     */
+    public static void AddImageFile(String filename) {
+        try {
+            Drawable image = ContextCompat.getDrawable(ApplicationProvider.getApplicationContext(),
+                    R.drawable.temp_diablerets);
+            assert image != null;
+            Bitmap bitmap = getBitmap(image);
+
+            File imageFile = new File(CameraActivity.getOutputDirectory(ApplicationProvider.getApplicationContext()),
+                    filename);
+            FileOutputStream outputStream = new FileOutputStream(imageFile);
+            int quality = 100;
```

Comment:
What if we make this parameter a final static attribute of the class `UITestHelper`? It would be easier to find and modify it, if necessary.

Reply:
In this case, I think it's fine if we keep it as it is.